### PR TITLE
Invoice

### PR DESCRIPTION
**Pull request by Pocketzs**
Completed invoice class

### CODE REVIEW

**Comment by KaraJoHo** [comment]
Looking good

File: lib/invoice.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    @status       = attributes[:status]
+    @created_at   = time_converter(attributes[:created_at])
+    @updated_at   = time_converter(attributes[:updated_at])
+  end
+
+  def time_converter(attributes)
+    return unless attributes
+    Time.parse(attributes)
+  end
+end
```
Comment (Pocketzs) [inline lib/invoice.rb]: Fixed

File: spec/invoice_spec.rb [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+      :merchant_id => 8,
+      :status      => "pending",
+      :created_at  => created = Time.now.to_s,
+      :updated_at  => updated = Time.now.to_s,
+    })
+
+    expect(i.id).to eq(6)
+    expect(i.customer_id).to eq(7)
+    expect(i.merchant_id).to eq(8)
+    expect(i.status).to eq("pending")
```
Comment (Pocketzs) [inline spec/invoice_spec.rb]: Fixed

**Review by beddings81 — APPROVED** [review]

**Review by KaraJoHo — APPROVED** [review]

**Review by Kledin85 — APPROVED** [review]
lookin good.

**Pull request merged**